app/src/main/java/com/napzak/market/presentation/explore/bottomSheet/SortBottomSheet.kt

```diff
@@ -0,0 +1,117 @@
+package com.napzak.market.presentation.explore.bottomSheet
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.dp
+import com.napzak.market.core.designsystem.theme.NapzakMarketTheme
+import com.napzak.market.presentation.explore.type.SortType
+import com.napzak.market.R
+import com.napzak.market.R.string.check_button
+import com.napzak.market.core.common.extension.noRippleClickable
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SortBottomSheet(
+    selectedSortType: String,
+    onDismissRequest: () -> Unit,
+    onSortItemClick: (String) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .background(NapzakMarketTheme.colors.black70)
+            .noRippleClickable {
+                onDismissRequest()
+            },
+    ) {
+        val sortList = listOf<SortType>(
+            SortType.RECENT,
+            SortType.POPULAR,
+            SortType.HIGH_PRICE,
+            SortType.LOW_PRICE,
+        )
+
+        Spacer(Modifier.weight(1f))
+
+        Column(
+            modifier = Modifier
+                .clip(RoundedCornerShape(topStart = 20.dp, topEnd = 20.dp))
+                .background(NapzakMarketTheme.colors.white)
+                .noRippleClickable { }
+                .padding(top = 20.dp)
+                .padding(horizontal = 20.dp),
+        ) {
+            sortList.forEach { sortItem ->
+                SortItem(
+                    sortType = sortItem.label,
+                    isSelected = sortItem.name == selectedSortType,
+                    isLastItem = sortItem.name == SortType.LOW_PRICE.name,
```

Comment:
p1: .name에 대해서 말씀드렸던 리뷰와 마찬가지로 이렇게 문자열로 직접 비교를 하는 것은 유지보수성과 안전성 측면에서 문제가 있을 수 있다고 생각합니다. 
```suggestion
                   isSelected = sortItem == SortType.valueOf(selectedSortType),
```
위와 같이 열거형 자체를 비교하는 방법도 있을 수 있고 enum에 id나 value 속성을 추가해서 사용하시는 것도 방법이 될 수 있을 것 같습니다!!